### DISCUSSION TITLE
Add possibility to change brightness via terminal command

### DISCUSSION THREAD

**Discussion by danilNikolaenko1990** [MonitorControl/MonitorControl#1602] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

It would be great to have the ability to adjust the brightness or set a specific level through the terminal using the same program. 

### Describe the solution you'd like

Since in some situations I want to adjust the brightness using Automator, adding it as one of the steps, and since there is already this program and I can’t just change the brightness with one command through bash, I need to install an additional CLI utility, which means I will have two utilities for adjusting brightness on my computer!

It would be great to enter moncontrol -set 100 or moncontrol -set 50 in the terminal, and maybe even moncontrol Up/Down.

One of my use cases is that through Automator, I can switch between dark and light themes with a hotkey, as well as enable energy-saving mode, and I would like to add fixed brightness adjustment to this scenario using your program. Thank you.

### Describe alternatives you've considered

Unfortunatelly I don't have another ideas

**Comment by waydabber** [comment, marked as answer]
Try using BetterDisplay and `betterdisplaycli` for this. CLI for MonitorControl is not likely to happen.

https://github.com/waydabber/BetterDisplay/wiki/Integration-features,-CLI
